packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -141,7 +141,7 @@ export interface VideoGalleryStyles extends BaseCustomStyles {
   verticalGallery?: VerticalGalleryStyles;
 }
 
-/* @conditional-compile-remove(vertical-gallery) */
+/* @conditional-compile-remove(vertical-gallery) */ /* @conditional-compile-remove(pinned-participants) */
```

Comment:
Why are we stabilizing the overflowGallery position here?

Reply:
removed